### PR TITLE
Add tests for utils

### PR DESCRIPTION
**Pull request by naveen521kk**
Just found that there was a style which we didn't export and we import it. Fixed it, and added tests so that it doesn't happen later.

### CODE REVIEW

**Comment by naveen521kk** [comment]
@PhilippImhof Can you review this?

**Comment by PhilippImhof** [comment]
Will do later today. Sorry for the delay.

File: tests/test_markup.py [hunk 1]
```diff
@@ -14,11 +18,14 @@ def test_good_markup():
...
     assert not manimpango.MarkupUtils.validate(
         "<b>foo"
     ), '"<b>foo" should fail validation (unbalanced tags)'
     assert not manimpango.MarkupUtils.validate(
         "<xyz>foo</xyz>"
     ), '"<xyz>foo</xyz>" should fail validation (invalid tag)'
+
+
+def test_markup_text():
+    MarkupText("<span>Hello Manim</span>")
```
Comment (PhilippImhof) [inline tests/test_markup.py:31]: I suggest a creating a text that actually uses some of the markup features

```suggestion
    MarkupText('<span underline="error"><b><i>Hello Manim</i></b></span>')
```

**Review by PhilippImhof — CHANGES_REQUESTED** [review]
Everything looks good; just a single nitpick for `test_markup_text`

**Review by PhilippImhof — APPROVED** [review]
LGTM

**Pull request merged**